pytest_motor/plugin.py

```diff
@@ -1,19 +1,19 @@
 """A pytest plugin which helps test applications using Motor."""
 import asyncio
-import platform
 import secrets
 import shutil
 import socket
-import tarfile
-import tempfile
 from pathlib import Path
-from typing import AsyncIterator, Iterator, List, Tuple
+from typing import AsyncIterator, Iterator, List
 
-import aiohttp
 import pytest
 from _pytest.config import Config as PytestConfig
 from motor.motor_asyncio import AsyncIOMotorClient
 
+from .mongod_binary import MongodBinary
```

Comment:
I prefer absolute instead of relative imports as the Google style guide recommends.